### ISSUE TITLE
Anyone has a desync chams code ?

### ISSUE THREAD

**Issue by neazen** [danielkrupinski/Osiris#498]
It would be cool adding a desync chams or if anyone has one for Osiris post it here ?

**Comment by effex1337** [comment]
Yes i have it the only reason I'm not sharing is the rapid attention osiris has been getting. 
It used to have like 60-70 in the issue tab and holy shit it's doubled in a few weeks. 
People have been claiming that the cheat is their own and they have self coded all of it. 
Some example foxware YouTube channel, several other renamed (p2c sites).
It's so sad people try to sell literally anything.

**Comment by neazen** [comment]
yeah, I myself posted something about that some time ago ![1](https://i.imgur.com/4j49DDc.png)

**Comment by Mufinekkk** [comment]
Anyone have that code? I really need that

**Comment by effex1337** [comment]
It's literally open src
You just need an iq greater than an ape to figure it out.

**Comment by tpincapps** [comment]
desync chams site:www.unknowncheats.me
p100 google search btw just sayin
